account_bank_statement_import_camt/camt.py

```diff
@@ -198,6 +182,7 @@ def parse_statement(self, ns, node):
         transaction_nodes = node.xpath('./ns:Ntry', namespaces={'ns': ns})
         for entry_node in transaction_nodes:
             transaction = statement.create_transaction()
+            transaction.data = etree.tostring(entry_node)
```

Comment:
Is this not redundant given that this attribute is also added in parse_transaction below?